index.js

```diff
@@ -82,6 +128,30 @@ module.exports = {
     return tree;
   },
 
+  postBuild({ directory }) {
+    if (!process.env.EMBER_CLI_ELECTRON) {
+      return tree;
+    }
+
+    //
+    // electron-forge needs our package.json to have a main entry pointing to
+    // electron.js. I can't figure out how to get broccoli to edit JSON for us,
+    // so we just do it manually once the build is complete.
+    //
```

Comment:
Ah, OK, I agree with your reasoning about not wanting to clutter a project's `package.json` with `ember-electron` configuration (whether that's the previously used `ember-electron` property with `copy-files` or the `main` property). In my case our apps using `ember-electron` don't have other deployment targets and aren't used by any other projects (shared parts are separated into their own modules). So it seems more complicated than necessary for those cases, but I can see where it might be useful in other situations.